spec/pubid_iso/identifier_spec.rb

```diff
@@ -215,64 +215,71 @@ module Pubid::Iso
       let(:pubid) { "ISO 10231:2003/Amd 1:2015" }
       let(:urn) { "urn:iso:std:iso:10231:amd:2015:v1" }
 
-      it_behaves_like "converts pubid to urn"
+      it "should raise an error when converting to URN" do
+        expect { subject.urn }.to raise_exception(Errors::NoEditionError)
+      end
+
       it_behaves_like "converts pubid to pubid"
     end
 
     context "ISO 10360-1:2000/Cor 1:2002" do
+      let(:original) { "ISO 10360-1:2000/Cor 1:2002 ED1" }
```

Comment:
@ronaldtse I added edition version to source identifiers, so don't have to remove these tests to check if URN building correctly.
I am uncertain if it's a good solution, but nothing better came to my mind.

Reply:
Seems fine for now. If we need further tests that don't have edition, the time will come.